examples/llama2/tensor_parallel_user_script.py

```diff
@@ -0,0 +1,134 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+from itertools import chain
+
+import torch
+import torch.nn.init
+from transformers import LlamaConfig
+
+
```

Comment:
Do we need these separate user script file when we already got https://github.com/microsoft/Olive/blob/main/examples/llama2/user_script.py ?

Reply:
Removed the alternate implementation.